docs/getting_started/discover_ui.md

```diff
@@ -4,22 +4,190 @@ description: Take a quick tour of the Back Office.
 
 # User interface
 
-## My dashboard
+## Dashboard
 
-The first screen that you see after logging in is **My dashboard**. 
-It contains shortcuts to most commonly used areas:
+The first screen that you see after logging in is **Dashboard**.
+It is default dashboard and contains following blocks:
 
-- review queue
-- your drafts
-- the content that you have created
-- recently modified content, etc.
+- **Quick actions** - displays most popular/used actions and shortcuts, for example, **Create Content**.
 
-You can select any of these items and view them in detail or start [editing them](create_edit_content_items.md) from this point.
+- **Recent activity** - a list of last activities (maximum 10), including timestamp, user data, and activity type.
+Available filters are: User(s), Activity type (create, update, publish, delete, hide, show) and Activity area (product, content).
+
+- **Ibexa news** - a list of recent blog posts or articles published at `ibexa.co` blog.
+It includes title, image, timestamp, and link to article details. List can containt minimum 1 and maximum 10 links.
+
+- **Review queue** - a list of Content items which user or User group can review.
+
+- **My content** - a list of Content items created by the user currently logged in.
+
+- **Common content** - a list of Content items created by all the organization that the user belongs to. You can select any of these items and view them in detail or start [editing them](create_edit_content_items.md) from this point.
 You can also [reschedule or cancel planned publications](../content_management/schedule_publishing.md#reschedule_or_cancel_publications) using the dashboard.
 
-![My dashboard](img/dashboard.png "My dashboard")
+![Dashboard](img/dashboard.png "Dashboard")
+
+From any page you can always return to the **Dashboard** screen by clicking the logo in the upper left corner.
+
+![Dashboard icon](img/dashboard_icon.png)
+
+## Customizable dashboard
+
+You can customize the dashboard depending on your needs.
+Customized dashboard previews a set of widgets selected by the user.
+
+It provides quick overview of these areas, that are commonly used, excluding those that are not used, or used rarely.
+This solution improves productivity, allows to enhance the default dashboard with additional widgets, and helps to make better business decisions based on data.
+
+By highlighting important information, users can work more efficiently with a clear view of tasks and deadlines.
+Customization includes also choosing one of predefined layouts and adding custom ones to use them in dashboards.
```

Comment:
How about changing the wording of this section and moving the remaining procedure to a separate article:

"Customization allows you to quickly visit the areas that are used more often and exclude those less important:
- Selected widgets boost productivity by highlighting vital data and helping in decision making
- Predefined and custom layouts improve visibility for efficient task and deadline management"

For more information, see "Work with dashboard".